OrbitGl/Track.cpp

```diff
@@ -150,8 +151,8 @@ void Track::Draw(GlCanvas* canvas, PickingMode picking_mode) {
     float label_offset_x = layout.GetTrackLabelOffsetX();
     float label_offset_y = layout.GetTrackLabelOffsetY();
     const Color kTextWhite(255, 255, 255, 255);
-    canvas->AddText(label_.c_str(), tab_x0 + label_offset_x, y1 + label_offset_y + m_Size[1],
-                    text_z, kTextWhite, label_width - label_offset_x);
+    canvas->AddText(label_.c_str(), tab_x0 + label_offset_x, toggle_y_pos - label_offset_y, text_z,
```

Comment:
I think this is abusing the TrackLabelOffset - in the end, you want the label to be vertically centered at the center of the triangle, and you're hardcoding this into the label offset. This will be quite unexpected if someone tries to adjust the layout. Can we easily get the height of the drawn label and then v-center it so this would also adjust to changed font sizes?

Reply:
TrackLabelOffsetY was only for that. I erased it and write something which is related with the font size. Now, if you change the font size is still aligned. BTW, this 3.5 isn't very nice, but is the best that I can find.

Anyway, I have a TODO about scaling the text and triangle sizes.

Reply:
I erased the TrackLabelOffsetY, and make something related with the font size. 

BTW, It's a temporal solution and didn't want to spend more time here. In the future we might want to modify font_size and triangle size when vertical-zooming. I didn't think too much how to modify track font-size without modifying the one in the TimeBar. Seems a little mess, but maybe it's easier than I expected.